cmd/stackdriver-prometheus-sidecar/main_test.go

```diff
@@ -171,3 +179,92 @@ func TestParseWhitelists(t *testing.T) {
 		})
 	}
 }
+
+func TestProcessFileConfig(t *testing.T) {
+	mustParseMetricSelector := func(input string) []*labels.Matcher {
+		if m, err := promql.ParseMetricSelector(input); err != nil {
+			t.Fatalf("bad test input %v: %v", input, err)
+			return nil
```

Comment:
Nit: Could omit the "return nil" and else.

Reply:
Done.